pkgs/development/libraries/libskk/default.nix

```diff
@@ -1,22 +1,22 @@
 { stdenv, fetchurl, fetchFromGitHub,
-  libtool, intltool, pkgconfig,
+  libtool, gettext, pkgconfig,
   vala, gnome_common, gobjectIntrospection,
-  libgee_0_8, json_glib, skk-dicts }:
+  libgee, json_glib, skk-dicts }:
 
 stdenv.mkDerivation rec {
   name = "libskk-${version}";
-  version = "1.0.2";
+  version = "1.0.3";
 
   src = fetchFromGitHub {
     owner = "ueno";
     repo = "libskk";
-    rev = "6a232e75de6d5dbe543ab17c9b85dc7560093509";
-    sha256 = "1xa9akf95jyi4laiw1llnjdpfq5skhidm7dnkd0i0ds6npzzqnxc";
+    rev = version;
+    sha256 = "092bjir866f350s4prq9q0yg34s91vmr8wbgf2vh3kcax1yj1axm";
   };
 
   buildInputs = [ skk-dicts ];
-  nativeBuildInputs = [ vala gnome_common gobjectIntrospection libtool intltool pkgconfig ];
-  propagatedBuildInputs = [ libgee_0_8 json_glib ];
+  nativeBuildInputs = [ vala gnome_common gobjectIntrospection libtool gettext pkgconfig ];
```

Comment:
just out of interest, when do I use `gettext`, when do I want `intltool`?

Reply:
`intltool` is basically abandoned now and `gettext` should be able to do all the things you needed `intltool` for. Though the helper commands are different so you can only drop intltools when a package does so (reading changelogs is useful). See also https://wiki.gnome.org/Initiatives/GnomeGoals/GettextMigration